.ci/docker/buildtools-windows/Dockerfile

```diff
@@ -6,21 +6,18 @@ FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-1909
 # Restore the default Windows shell for correct batch processing.
 SHELL ["cmd", "/S", "/C"]
 
-# Download the Build Tools bootstrapper.
-ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\TEMP\vs_buildtools.exe
+# Download NuGet v5.4.0 
+ADD https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe C:\nuget\nuget.exe
 
-# Install Build Tools excluding workloads and components with known issues.
-RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache `
-    --installPath C:\BuildTools `
-    --all `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10240 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10586 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.14393 `
-    --remove Microsoft.VisualStudio.Component.Windows81SDK `
- || IF "%ERRORLEVEL%"=="3010" EXIT 0
+# Add NuGet to Beginning of Path (We want this version of Nuget before any other versions that may be in the path)
+RUN @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[Environment]::SetEnvironmentVariable(\"Path\", \"C:\nuget;\" + $env:Path, [EnvironmentVariableTarget]::Machine)"
```

Comment:
Could this particular command simplified with the below suggestion?

```suggestion
##  Set the path
RUN setx /M PATH "C:\nuget;%PATH%;"
```

Refers to https://github.com/PowerShell/PowerShell-Docker/blob/master/release/preview/windowsservercore/docker/Dockerfile#L49

Reply:
I was having problems getting setx to stick will try this again.